azure/table_azure_api_management_backend.go

```diff
@@ -0,0 +1,225 @@
+	serviceInfo := h.Item.(apimanagement.ServiceResource)
+	serviceName := *serviceInfo.Name
+	resourceGroup := strings.Split(*serviceInfo.ID, "/")[4]
+
+	session, err := GetNewSession(ctx, d, "MANAGEMENT")
+	if err != nil {
+		return nil, err
+	}
+	subscriptionID := session.SubscriptionID
+
+	apiManagementBackendClient := apimanagement.NewBackendClientWithBaseURI(session.ResourceManagerEndpoint, subscriptionID)
+	apiManagementBackendClient.Authorizer = session.Authorizer
+
+	result, err := apiManagementBackendClient.ListByService(ctx, resourceGroup, serviceName, "", nil, nil)
+	if err != nil {
+		plugin.Logger(ctx).Error("listAPIManagementBackends", "list", err)
+		return nil, err
+	}
+	for _, apiManagementBackend := range result.Values() {
+		backendWithService := &BackendWithServiceName{
+			apiManagementBackend,
+			serviceName,
+		}
+		d.StreamListItem(ctx, backendWithService)
+		// Check if context has been cancelled or if the limit has been hit (if specified)
+		// if there is a limit, it will return the number of rows required to reach this limit
+		if d.RowsRemaining(ctx) == 0 {
+			return nil, nil
+		}
+	}
+
+	for result.NotDone() {
+		err = result.NextWithContext(ctx)
+		if err != nil {
+			plugin.Logger(ctx).Error("listAPIManagementBackends", "list_paging", err)
+			return nil, err
+		}
+
+		for _, apiManagementBackend := range result.Values() {
+			d.StreamListItem(ctx, apiManagementBackend)
+			// Check if context has been cancelled or if the limit has been hit (if specified)
+			// if there is a limit, it will return the number of rows required to reach this limit
+			if d.RowsRemaining(ctx) == 0 {
+				return nil, nil
+			}
+		}
+	}
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getAPIManagementBackend(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Debug("getAPIManagementBackend")
+
+	backendID := d.EqualsQualString("backend_id")
+	serviceName := d.EqualsQualString("service_name")
+	resourceGroup := d.EqualsQualString("resource_group")
+
+	// resourceGroupName can't be empty
+	// Error: pq: rpc error: code = Unknown desc = apimanagement.ServiceClient#Get: Invalid input: autorest/validation: validation failed: parameter=serviceName
+	// constraint=MinLength value="" details: value length must be greater than or equal to 1
+	if len(backendID) < 1 {
+		return nil, nil
+	}
+
+	session, err := GetNewSession(ctx, d, "MANAGEMENT")
+	if err != nil {
+		return nil, err
+	}
+	subscriptionID := session.SubscriptionID
+
+	apiManagementBackendClient := apimanagement.NewBackendClientWithBaseURI(session.ResourceManagerEndpoint, subscriptionID)
+	apiManagementBackendClient.Authorizer = session.Authorizer
+
+	op, err := apiManagementBackendClient.Get(ctx, resourceGroup, serviceName, backendID)
+	if err != nil {
+		plugin.Logger(ctx).Error("getAPIManagementBackend", "get", err)
```

Comment:
Please update the log statements per the Steampipe standard(`"<table name>.<function name>", "client/api error", err`)?